operatorcourier/push.py

```diff
@@ -28,6 +29,7 @@ def push(self, bundle_dir, namespace, repository, release, auth_token):
         :param auth_token: Authentication token used to push to Quay.io.
         """
         logger.info('Generating 64 bit bundle and pushing to app registry.')
+        filterOutFiles(bundle_dir, BLACK_LIST)
```

Comment:
Note that this removes files from a temporary directory created at runtime (not the actual directory)